### ISSUE TITLE
nuisflat with hepmc file does not get the correct cross section for fScaleFactor occasionally

### ISSUE THREAD

**Issue by JakeMcKean-code** [NUISANCEMC/nuisance#66]
When using `nuisflat` with some GENIE/Achilles files, the "`totXS`" is sometimes not taken correctly from the hepmc file and a different value is put into the `fScaleFactor` in the flattree. 
Verified by manually checking the hepmc file and extracting the `totXS` value and this was more sensible. Value in `fScaleFactor` was different (by not just a multiple of 10)

**Comment by luketpickering** [comment]
hmm, what is 'totXS' here?

Can you attach the first few events from an example file here and I will try and reproduce.

I suspect it is trying to swap from per nulceon to per target, which is wrong and I need to fix.

**Comment by JakeMcKean-code** [comment]
Yes the totXS is what is in the HepMC3 file, this was different to the fScaleFactor that nuisflat was producing. When using the GENIE ROOT file output instead, the issue was not there. I've attached a zip of the hepmc3 file containing just the first few events. 
[nueb_on_1000010010_500MeV_0_first_few_events.hepmc3.zip](https://github.com/user-attachments/files/17291957/nueb_on_1000010010_500MeV_0_first_few_events.hepmc3.zip)
